### ISSUE TITLE
Passwords autofill catalog not up to date

### ISSUE THREAD

**Issue by AiroPi** [johannes-schliephake/nextcloud-passwords-ios#276]
### iOS App Version

2.7.1 (TestFlight Build 41)

### iOS Version

17.1.2

### Device Model

iPhone 15

### Observed Behavior

When I use the autofill feature in any app, the list doesn’t show the passwords added after the initial connection. These passwords are available in the app. They have mainly been created using the app.

### Expected Behavior

To have the same passwords in the autofill menu than in the app.

### Debugging

- [X] I did install the [TestFlight](https://testflight.apple.com/join/iuljLJ4u) version to check the log on the app's *Settings* page → *Log*.
- [X] I did check for iOS logs named *Passwords* under *System Settings* → *Privacy & Security* → *Analytics & Improvements* → *Analytics Data*.
- [X] I did check for Nextcloud logs on the admin's *Logging* page.

**Comment by johannes-schliephake** [comment]
Hey, thanks for reporting this. Just to be sure: We are probably talking about the buttons shown above the keyboard, right?

**Comment by AiroPi** [comment]
Yes, or when we long press a text input, we can choose "Autofill" then "password" and the Passwords app will appear as a "popup".
